components/Editor/index.tsx

```diff
@@ -238,6 +238,83 @@ const Editor = ({ readOnly = false }: Props) => {
   ]
   useRegisterActions(actions, [highlightCode])
 
+  const handleCommentLine = useCallback(() => {
+    if (!editorRef.current) {
+      return
+    }
+    const textareaRef = editorRef.current._input
+    const lineNumberStart = textareaRef.value
```

Comment:
would be better to rename to `selectionLineNumberStart` and `selectionLineNumberEnd` for clarity